### PR TITLE
Change msbuild error

### PR DESCRIPTION
**Pull request by Doprez**
# PR Details

This PR should make the readabilty of a common error easier to understand.

## Description

I added references to the .NET 6 package in 2 places where an error describes needing MSBuild 16.0. I added these because even the VS2022 installation does not solve this error and user need to manually install the .NET 6 SDK in order for gamestudio 4.1 to run.

I also did a little bit of code clean up with some if statements.

## Related Issue

https://github.com/stride3d/stride/issues/1612
also at least one question about this problem a week in the discord.

## Motivation and Context

There are a lot of new Stride user coming in that have not manually installed the .NET 6 SDK. I am hoping this error message will be a bit more friendly to read. 

## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by Doprez** [comment]
> 

@dotnet-policy-service agree

File: sources/assets/Stride.Core.Assets/PackageSessionPublicHelper.cs [hunk 1]
```diff
@@ -41,13 +41,19 @@ public static void FindAndSetMSBuildVersion()
                     ? x.DiscoveryType == DiscoveryType.DotNetSdk && x.Version.Major >= 3
                     : (x.DiscoveryType == DiscoveryType.VisualStudioSetup || x.DiscoveryType == DiscoveryType.DeveloperConsole) && x.Version.Major >= 16);
 
+                if (MSBuildInstance == null)
+                {
+                    throw new InvalidOperationException("Could not find a MSBuild installation (expected 16.0 or later) " +
+                        "Please ensure you have the .NET 6 SDK installed from Microsofts Website");
```
Comment (manio143) [inline sources/assets/Stride.Core.Assets/PackageSessionPublicHelper.cs:47]: Nitpick: "Microsoft's" (with apostrophe) and "website" shouldn't be capitalized. Ideally a link to https://dotnet.microsoft.com/ should be used instead.

**Review by manio143 — COMMENTED** [review]
Looks good. Please squash your changes and rebase your branch to remove commits unrelated to this PR.

File: sources/assets/Stride.Core.Assets/PackageSessionPublicHelper.cs (hunk 1 above)
Comment (Doprez) [reply sources/assets/Stride.Core.Assets/PackageSessionPublicHelper.cs]: should be updated now.

**Comment by Doprez** [comment]
going to remake this PR because Im bad at git.

**Comment by Doprez** [comment]
> Looks good. Please squash your changes and rebase your branch to remove commits unrelated to this PR.

https://github.com/stride3d/stride/pull/1643 new PR because I totally messed up my master branch.

**Pull request closed without merging**